.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,17 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Start a discussion
+    url:  https://github.com/FreeTubeApp/FreeTube/discussions/categories/general
+    about: Feel free to start a discussion
+  - name: Ask a question
+    url:  https://github.com/FreeTubeApp/FreeTube/discussions/categories/q-a
+    about: Ask and answer questions
+  - name: Join our Matrix Community
+    url: https://matrix.to/#/+freetube:matrix.org
+    about: "Note: Bugs and Feature requests should be made on GitHub and not in the Matrix room."
+  - name: Help translate FreeTube on Weblate
+    url: https://hosted.weblate.org/engage/free-tube/
+    about: Help translate FreeTube on Weblate
+  - name: View the FreeTube docs
+    url: https://docs.freetubeapp.io/
+    about: View the FreeTube docs
```

Comment:
```suggestion
    about: View the Documentation for the recording of any and all relevant information about FreeTube.
```

Reply:
@ChunkyProgrammer noticed that this one wasnt part of your commit. Was this a mistake or do you think this can be worded better?

Reply:
I find the wording to be a bit confusing but I'm not sure what could make it better exactly (the part that I find confusing is `the recording of any and all`)

Reply:
Hmm i understand, what about

`View the Documentation to find all relevant information about FreeTube`